perception/bytetrack/src/bytetrack_node.cpp

```diff
@@ -83,10 +83,18 @@ void ByteTrackNode::on_rect(
   bytetrack::ObjectArray objects = bytetrack_->update_tracker(object_array);
   for (const auto & tracked_object : objects) {
     tier4_perception_msgs::msg::DetectedObjectWithFeature object;
-    object.feature.roi.x_offset = tracked_object.x_offset;
-    object.feature.roi.y_offset = tracked_object.y_offset;
-    object.feature.roi.width = tracked_object.width;
-    object.feature.roi.height = tracked_object.height;
+    // fit xy offset to 0 if roi is outside of image
+    const int outside_x = 0 - tracked_object.x_offset;
+    const int outside_y = 0 - tracked_object.y_offset;
```

Comment:
[Q]
` std::max(-tracked_object.x_offset, 0)` is more understandable?

Reply:
I agree. Fixed in the last commit.